src/components/views/Page.tsx

```diff
@@ -59,16 +68,43 @@ const PageView = ({ children, title, breadcrumbs, actions }: PageViewProps) => {
           </Text>
           {actions && (
             <div className="flex gap-3">
-              {actions.map((action, index) => (
-                <Button
-                  key={`action-${index}`}
-                  onClick={action.onClick}
-                  loading={action.loading}
-                  size="sm"
-                >
-                  {action.content}
-                </Button>
-              ))}
+              <div className="md:hidden relative">
+                <DropdownMenu>
+                  <DropdownMenuTrigger asChild>
+                    <button className="inline-flex items-center justify-center whitespace-nowrap rounded-md focus-visible:outline-none border h-8 w-8 p-0">
```

Comment:
Is look of the button the problem here or simply the use of the `button` tag? Would this be better practice to do this using a div like so:
```
<DropdownMenuTrigger asChild>
     <div className="inline-flex items-center justify-center whitespace-nowrap rounded-md focus-visible:outline-none border h-8 w-8 p-0">
          <DotsVerticalIcon/>
     </div>
</DropdownMenuTrigger>
```

I used a button here for 3 reasons:
- Reason #1: In the code samples you sent me from shadCN they included a button in all of their drop down menu examples. Since you asked that I follow that shadCN drop down demo I figure you either may have liked the implementation of the the button element or that possible it implemented as a form of good practice.
- Reason #2: Is because the "next page" arrow at the bottom of the student table was implemented using a button. I found the code for this "arrow" button and copied almost the exact same thing to use for this "..." button to insure my code stayed consistent with the rest of the frontend code being used.
- Reason #3: I used the "button look" over simply using the "..." icon as I felt as though it not only looked much better and more consistent with the site but also because it increased the visibility and intuitive usability of the button.

Reply:
Oh no I mean like `<Button />` vs `<button />` - i.e. there is a defined button component in the repo

Reply:
Ohhhh I didn't even notice this 😅. That makes a lot more sense in that case. Now should this `<DotsVerticalIcon/>` be enclosed with `<Button />` or enclosed with `<div>`?

Reply:
The former, I think